Testing System/src/main/java/com/epam/practice/testingsystem/data/DataConnection.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.practice.testingsystem.data;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataConnection {
+    public static Connection getConnection() {
+        String url = System.getenv("DB_URL");
+        String user = System.getenv("DB_USER");
+        String password = System.getenv("DB_PASSWORD");
+
+        try {
+            return DriverManager.getConnection(url, user, password);
+        }
+        catch (SQLException e) {
+            e.printStackTrace();
+            return null;
```

Comment:
Partially fixed. Now `null` values are used only as a response from `find(int id)` methods if there is no entry in the table with specified id. Otherwise a `RuntimeError` is thrown.